hls4ml/templates/quartus/firmware/nnet_utils/nnet_embed_stream.h

```diff
@@ -0,0 +1,10 @@
+/*
+*   PLACEHOLDER - The common pass embedding.py includes both parallel and streaming implementations; streaming is currently not supported in Quartus
+*/
+
+#ifndef NNET_BATCHNORM_STREAM_H_
+#define NNET_BATCHNORM_STREAM_H_
+
+namespace nnet {}
```

Comment:
Can we add a stub that raises an exception while we still don't have the implementation? Otherwise the compilation fails with a cryptic message (cannot substitute template argument or something like that). Or does that require the more streaming infrastructure in Quartus (#557)?

Reply:
This file should never be invoked, as soon as Quartus attempts to use io_stream an exception is thrown from Python (although https://github.com/fastmachinelearning/hls4ml/pull/557 attempts to add infrastructure for streaming, which means for some layers we should throw exceptions and for some not - this should be handled by Python and can be a part of the streaming PR)